integration_testing/features/super-admin/super-admin-export-data.feature

```diff
@@ -12,18 +12,12 @@ Feature: Super admin exports usage data
       And the *Download* button is enabled
       And the text change to "Generate a new log file" 
     When I click on *Download* button     
-    Then I see *Save as* window
-    When I enter the name and location for the CSV file
-      And I click the *Save* button
-    Then I see the session logs file saved on my local drive
+    Then I see that a file named content_session_logs.csv is automatically saved on my local drive
```

Comment:
This actually depends on what the user has as the default browser behavior, so let's integrate this into:

```Then I see the *Open/Save as* window, or the file 'content_session_logs.csv' is automatically saved on my local drive, depending on the browser defaults```